LAYOUT_STYLE_GUIDE.md

```diff
@@ -545,3 +545,13 @@ While Markdown supports the use of both asterisks `*` and underscores `_` to mak
 
 __This__ is _not_ how we do things.
 ```
+
+If asterisks are needed for showing multiplication, wrap the math expression in single backticks to form [inline code](#inline).
+
+```markdown
+Correct:
+`4 * 3 * 2`
+
+Incorrect:
+4 * 3 * 2
+```
```

Comment:
Kinda feel like rather than calling out specifically this case here with such attention, we should maybe just call out any math formula/calculation in the code block section. "Use them for short code quotations, field names, math calculations, or file names."

With the linter catching this kinda stuff, I think there is less of a need to call out super specific scenarios like this with such attention.

Reply:
That's ... a much better idea. I'll get it changed over :sweat_smile: